newIDE/app/src/GameDashboard/Marketing/MarketingPlans.js

```diff
@@ -0,0 +1,478 @@
+                              <Line noMargin alignItems="center">
+                                <CheckCircleFilled
+                                  style={{
+                                    ...styles.bulletIcon,
+                                    ...(activeFeaturing
+                                      ? { color: gdevelopTheme.message.valid }
+                                      : {}),
+                                  }}
+                                />
+                                <Text style={{ flex: 1 }}>
+                                  {selectMessageByLocale(
+                                    i18n,
+                                    bulletPointByLocale
+                                  )}
+                                </Text>
+                              </Line>
+                            </Column>
+                          )
+                        )}
+                      </div>
+
+                      <Column
+                        noMargin
+                        alignItems="flex-start"
+                        expand
+                        justifyContent="flex-end"
+                      >
+                        <Text
+                          size="body-small"
+                          noMargin
+                          color="secondary"
+                          align="left"
+                        >
+                          {activeFeaturing ? (
+                            activeFeaturing.featuring ===
+                            'games-platform-home' ? (
+                              <Trans>
+                                Active until{' '}
+                                {i18n.date(activeFeaturing.expiresAt * 1000)}
+                              </Trans>
+                            ) : (
+                              <Trans>
+                                Active, we will get in touch to get the campaign
+                                up!
+                              </Trans>
+                            )
+                          ) : (
+                            selectMessageByLocale(i18n, descriptionByLocale)
+                          )}
+                        </Text>
+                      </Column>
+                      <RaisedButton
+                        primary={!activeFeaturing}
+                        onClick={() => onPurchase(i18n, marketingPlan)}
+                        label={
+                          isLoading ? (
+                            <Column
+                              noMargin
+                              justifyContent="center"
+                              alignItems="center"
+                            >
+                              <CircularProgress size={20} />
+                            </Column>
+                          ) : activeFeaturing ? (
+                            activeFeaturing.featuring ===
+                            'games-platform-home' ? (
```

Comment:
Should this hardcoded value be here?

Reply:
This prop is supposed to be typed yeah, maybe I should export it from somewhere else